keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -931,6 +936,7 @@ Pick one more of these and delete the rest.
   - Metric name: apiserver_cache_watcher_buffer_length (histogram, what was the buffer size)
   - Metric name: apiserver_watch_cache_lag (histogram, for how far the cache is behind the expected RV)
   - Metric name: apiserver_terminated_watchers_total (counter, already defined, needs to be updated (by an attribute) so that we count closed watch requests due to an overfull buffer in the new mode)
+  - Metric name: apiserver_watch_cache_watch_list_duration_seconds (histogram, measures latency of watch-list requests)
```

Comment:
Let's not add it now - I'm not convinced about this metric - see my comments on the PR that adds it.